src/utils/checkDate.ts

```diff
@@ -0,0 +1,8 @@
+const dateInPast = (userDate: string): boolean => {
+  const [day, month, year] = userDate.split("/").map(Number);
```

Comment:
I think the date could just be a DateTime string and avoid doing this if it is formatted 'YYYY-MM-DD'('2000-01-31') in the spreadsheet. Then you can parse it directly into the Date class

Reply:
I think that due to the rudimentary design of our db, it is better to allow the user to input date in a format that they are used to, and parse it on our end, to avoid any errors

Reply:
YYYY-MM-DD is not an unusual format though for dates, its actually the standard but ok! :)